docs/content/app-protect-waf/installation.md

```diff
@@ -60,7 +60,39 @@ Take the steps below to set up and deploy the NGINX Ingress Controller and App P
     > **Important**: You must have an admin role to configure RBAC in your Kubernetes cluster.
 
 2. [Create the common Kubernetes resources](/nginx-ingress-controller/installation/installation-with-manifests/#2-create-common-resources).
+
 3. Enable the App Protect WAF module by adding the `enable-app-protect` [cli argument](/nginx-ingress-controller/configuration/global-configuration/command-line-arguments/#cmdoption-enable-app-protect) to your Deployment or DaemonSet file.
+
+If you intend to use the F5 NGINX Ingress Controller with the AppProtect WAF module and policy bundles, follow these additional steps:
```

Comment:
```suggestion
If you intend to use the NGINX Ingress Controller with the AppProtect WAF module and policy bundles, follow these additional steps:
```

No need to prepend F5 in the body text; it's something that only applies to the first instance of the product name, usually near the top of the page.